README.md

```diff
@@ -80,4 +80,27 @@ client.is_enabled("My Toggle", fallback_function=custom_fallback)
 
 - Must accept the fature name and context as an argument.
 - Client will evaluate the fallback function once per call of `is_enabled()`.  Please keep this in mind when creating your fallback function!
-- If both a `default_value` and `fallback_function` are supplied, client will define the default value by `OR`ing the default value and the output of the fallback function.
\ No newline at end of file
+- If both a `default_value` and `fallback_function` are supplied, client will define the default value by `OR`ing the default value and the output of the fallback function.
+
+### Getting a varient
```

Comment:
*varient* typo in multiple places in this file.

Reply:
Oops, fixed! ><  It's not like I didn't spell it correctly everywhere else.